[2024-08-28T04:10:13.182+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-28T04:10:13.353+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: movies_dag.subir_data manual__2024-08-28T04:10:00.012561+00:00 [queued]>
[2024-08-28T04:10:13.362+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: movies_dag.subir_data manual__2024-08-28T04:10:00.012561+00:00 [queued]>
[2024-08-28T04:10:13.363+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 4
[2024-08-28T04:10:13.376+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): subir_data> on 2024-08-28 04:10:00.012561+00:00
[2024-08-28T04:10:13.383+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=110) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-28T04:10:13.384+0000] {standard_task_runner.py:72} INFO - Started process 112 to run task
[2024-08-28T04:10:13.385+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'movies_dag', 'subir_data', 'manual__2024-08-28T04:10:00.012561+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl.py', '--cfg-path', '/tmp/tmp0a__8iv9']
[2024-08-28T04:10:13.387+0000] {standard_task_runner.py:105} INFO - Job 37: Subtask subir_data
[2024-08-28T04:10:13.427+0000] {task_command.py:467} INFO - Running <TaskInstance: movies_dag.subir_data manual__2024-08-28T04:10:00.012561+00:00 [running]> on host e1752bf327c8
[2024-08-28T04:10:13.505+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='pedro' AIRFLOW_CTX_DAG_ID='movies_dag' AIRFLOW_CTX_TASK_ID='subir_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-28T04:10:00.012561+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-28T04:10:00.012561+00:00'
[2024-08-28T04:10:13.506+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-28T04:10:13.518+0000] {dag_intermediario.py:48} ERROR - ('Error when uploading data: ', TypeError("'NoneType' object is not subscriptable"))
[2024-08-28T04:10:13.518+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-08-28T04:10:13.528+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-28T04:10:13.529+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=movies_dag, task_id=subir_data, run_id=manual__2024-08-28T04:10:00.012561+00:00, execution_date=20240828T041000, start_date=20240828T041013, end_date=20240828T041013
[2024-08-28T04:10:13.559+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-28T04:10:13.576+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-28T04:10:13.578+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
